extensions/amp-story/0.1/amp-story.js

```diff
@@ -354,6 +354,18 @@ export class AmpStory extends AMP.BaseElement {
       this.ampStoryHint_.showFirstPageHintOverlay();
     });
 
+    this.element.addEventListener(EventType.TAP_NAVIGATION, e => {
+      const {navigationDirection} = e.detail;
+      if (navigationDirection === NAVIGATION_DIRECTION.NEXT) {
```

Comment:
1. Remove `navigation` as a prefix (i.e. `direction` is fine)
2. Short-circuit classes for special cases are always better.

```javascript
if (this.isDesktop_()) {
  this.next_();
  return;
}

if (direction == TapNavigationDirection.PREVIOUS) {
  this.previous_();
} else {
  this.next_():
}
```

Reply:
just curious why is one better that other?
i know the care where the else is huge block and gets intended, but why otherwise?

Reply:
Done

Reply:
Yeah, it's more readable as you mentioned. It also makes a clearer change if it ever gets removed in the future.